CodeStarterCamp_Week2/main.swift

```diff
@@ -7,31 +7,39 @@
 //
 
 import Foundation
+
 let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
-func createLottoNumbers() -> Set<Int> {
+var lottoWinNumbersDictionary: Dictionary<String, Array<Int>> = [:]
+
+
+func createLottoNumbers() -> [Int] {
     var lottoNumber: Set<Int> = []
     while lottoNumber.count < 6 {
         lottoNumber.insert(Int.random(in:1...45))
     }
-    print("이번 주 로또 번호 :",lottoNumber)
-    return lottoNumber
+    return lottoNumber.sorted()
 }
 
-func compareLottoNumbers(lottoWinNumbers: Set<Int>, mine: [Int]) {
-    var sameLottoNumbers = Set<Int>()
-    var sameLottoNumbersChangeString = String()
-    sameLottoNumbers = lottoWinNumbers.intersection(mine)
-    if sameLottoNumbers.count > 0 {
-        for stringCount in sameLottoNumbers {
-            sameLottoNumbersChangeString += String(stringCount) + ","
-        }
-        sameLottoNumbersChangeString.removeLast()
-        print("축하합니다! 겹치는 번호는 \(sameLottoNumbersChangeString)입니다!")
+func createRoundLottoNumbers(roundCount: Int) {
+    for round in 1...roundCount {
+        saveRoundLottoNumbers(storageRound: round, roundWinLottoSet: createLottoNumbers())
+    }
+}
+
+func saveRoundLottoNumbers(storageRound: Int, roundWinLottoSet: Array<Int>) {
```

Comment:
좀 더 좋은 네이밍이군요! 
고민하신 흔적이 보여서 좋습니다! 👍
다만, `roundWinLottoSet`과 같이 특정 타입을 변수나 함수명으로 지정하는것은 좋은 방법은 아닙니다 😅
아규먼트 레이블을 한번 활용해보는건 어떨까요?

정답은 절대 아니고(저도 영작은 잘 못합니다 😅)
저였다면 요렇게 매개변수의 위치와 아규먼트 레이블을 활용해서 함수명을 지어볼 거 같아요.
네이밍이 어려우실 때 좋은 팁은, 호출할 때 하나의 문장이 되도록 해보시면 좋습니다.

```suggestion
func save(lottoNumbers: Array<Int>, in round: Int) {
```

그러면 호출할 때는 다음과 같이 호출할 수 있겠네요.

```swift
let winNumbers = createLottoNumbers()

save(lottoNumbers: winNumbers, in: round)
```

Reply:
오호?? 이런 방법도 있군요?? 원비 덕분에 또 하나의 방법을 알게된 것 같습니다. 감사합니다^^ 다음에 적용할 수 있으면 한번 적용해봐야 될 것 같습니당~